src/main/java/com/aws/greengrass/dependency/ComponentStatusCode.java

```diff
@@ -15,31 +15,61 @@
 public enum ComponentStatusCode {
 
     NONE(""),
-    INSTALL_ERRORED("Error during install"),
-    INSTALL_INVALID_CONFIG("Invalid configuration for install"),
-    INSTALL_IO_ERROR("I/O error during install"),
-    INSTALL_MISSING_DEFAULT_RUNWITH("Could not determine user/group to run with during install"),
-    INSTALL_TIMEOUT("Timeout during install"),
-    STARTUP_ERRORED("Error during startup"),
-    STARTUP_INVALID_CONFIG("Invalid configuration for startup"),
-    STARTUP_IO_ERROR("I/O error during startup"),
-    STARTUP_MISSING_DEFAULT_RUNWITH("Could not determine user/group to run with during startup"),
-    STARTUP_TIMEOUT("Timeout during startup"),
-    RUN_ERRORED("Error during run"),
-    RUN_MISSING_DEFAULT_RUNWITH("Could not determine user/group to run with during run"),
-    RUN_INVALID_CONFIG("Invalid configuration for run"),
-    RUN_IO_ERROR("I/O error during run"),
-    RUN_TIMEOUT("Timeout during run"),
-    SHUTDOWN_ERRORED("Error during shutdown"),
-    SHUTDOWN_TIMEOUT("Timeout during shutdown");
+    INSTALL_ERROR("An error occurred during installation.",
```

Comment:
where are we interpolating the string?

Also, no period.

Reply:
These aren't log messages though